File: task/src/main/java/org/kharitonov/springWeb/config/SpringWebConfig.java
package org.kharitonov.springWeb.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;

@Configuration
@EnableWebMvc
@ComponentScan("org.kharitonov.springWeb")
public class SpringWebConfig {
}

File: laba_chetyre/src/main/kotlin/by/bsut/labachetyre/config/SpringWebConfig.kt
package by.bsut.labachetyre.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;

@Configuration
@EnableWebMvc
@ComponentScan("by.bsut.labachetyre")
public class SpringWebConfig